vendors/microchip/boards/ecc608a_plus_winsim/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ afr_mcu_port(kernel)
 
 # ECC608A source files
 afr_glob_src(ecc608a_src DIRECTORY ${ecc608a_dir}/lib)
+list(REMOVE_ITEM ecc608a_src "${ecc608a_dir}/lib/atca_cfgs.c")
```

Comment:
Can you add a comment here so that we do not forget and other people understand why we're doing this?

Reply:
Done